ngraph/core/include/ngraph/op/util/multi_subgraph_base.hpp

```diff
@@ -0,0 +1,389 @@
+                    ///
+                    /// \param      input_index           Position of the MultiSubGraphOp input
+                    /// \param      body_parameter_index  Body parameter to receive input
+                    ///
+                    InvariantInputDescription(uint64_t input_index, uint64_t body_parameter_index);
+                    InvariantInputDescription() = default;
+                    std::shared_ptr<InputDescription> copy() const override;
+                };
+
+                /// \brief Produces an output
+                class NGRAPH_API BodyOutputDescription : public OutputDescription
+                {
+                public:
+                    static constexpr type_info_t type_info{"BodyOutputDescription", 0};
+                    const type_info_t& get_type_info() const override { return type_info; }
+                    ///
+                    /// \brief      Constructs a new instance.
+                    ///
+                    /// \param      body_value_index  A body value that produces the output
+                    /// \param      output_index      The MultiSubGraphOp output index
+                    ///
+                    BodyOutputDescription(uint64_t body_value_index, uint64_t output_index);
+                    BodyOutputDescription() = default;
+                    std::shared_ptr<OutputDescription> copy() const override;
+                };
+                using MultiSubgraphInputDescriptionPtr =
+                    std::shared_ptr<MultiSubGraphOp::InputDescription>;
+                using MultiSubgraphOutputDescriptionPtr =
+                    std::shared_ptr<MultiSubGraphOp::OutputDescription>;
+                using MultiSubgraphInputDescriptionVector =
+                    std::vector<MultiSubgraphInputDescriptionPtr>;
+                using MultiSubgraphOutputDescriptionVector =
+                    std::vector<MultiSubgraphOutputDescriptionPtr>;
+
+                /// \brief     Gets internal sub-graph by index in MultiSubGraphOp
+                ///
+                /// \param     index sub-graph's index in op
+                /// \return pointer to ngraph::Function with sub-graph
+                virtual std::shared_ptr<Function> get_function(int index)
+                {
+                    return m_bodies[index];
+                };
+                /// \brief     Gets internal sub-graph by index in MultiSubGraphOp
+                ///
+                /// \param     index sub-graph's index in op
+                /// \return pointer to ngraph::Function with sub-graph
+                virtual std::shared_ptr<const Function> get_function(int index) const
+                {
+                    return m_bodies[index];
+                };
+                /// \brief     Adds sub-graph to MultiSubGraphOp
+                ///
+                /// \param     func new sub_graph as ngraph::Function
+                virtual void add_function(const std::shared_ptr<Function>& func)
```

Comment:
We need to double check that TI interface doesn't have this methods in public API

Reply:
this method was deleted.